File: test/org/rust/lang/core/parser/RustCompleteParsingTestCase.java
package org.rust.lang.core.parser;

public class RustCompleteParsingTestCase extends RustParsingTestCaseBase {

    public RustCompleteParsingTestCase() {
        super("parser/well-formed");
    }

    public void testFn()             { doTest(true); }
    public void testExpr()           { doTest(true); }
    public void testMod()            { doTest(true); }
    public void testUseItem()        { doTest(true); }
    public void testType()           { doTest(true); }
    public void testShifts()         { doTest(true); }
    public void testPatterns()       { doTest(true); }
    public void testAttributes()     { doTest(true); }
    public void testTraits()         { doTest(true); }
    public void testMacros()         { doTest(true); }
    public void testImpls()          { doTest(true); }
    public void testSuper()          { doTest(true); }
    public void testRanges()         { doTest(true); }
}

File: test/org/rust/lang/core/parser/RustCompleteParsingTestCase.kt
package org.rust.lang.core.parser

class RustCompleteParsingTestCase : RustParsingTestCaseBase("parser/well-formed") {


    // @formatter:off
    fun testFn()                    { doTest(true) }
    fun testExpr()                  { doTest(true) }
    fun testMod()                   { doTest(true) }
    fun testUseItem()               { doTest(true) }
    fun testType()                  { doTest(true) }
    fun testShifts()                { doTest(true) }
    fun testPatterns()              { doTest(true) }
    fun testAttributes()            { doTest(true) }
    fun testTraits()                { doTest(true) }
    fun testMacros()                { doTest(true) }
    fun testImpls()                 { doTest(true) }
    fun testSuper()                 { doTest(true) }
    fun testRanges()                { doTest(true) }
    fun testExternCrates()          { doTest(true) }
    // @formatter:off
}
